<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="Header">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QFrame" name="frame_logo">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>70</height>
         </size>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <widget class="QLabel" name="logo">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>0</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Logo/Images/paypal.svg&quot; height=&quot;50&quot; width=&quot;50&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="logoTitle">
         <property name="geometry">
          <rect>
           <x>55</x>
           <y>5</y>
           <width>100</width>
           <height>38</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Inter</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>PSWRD
MANAGER</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <zorder>logoTitle</zorder>
        <zorder>logo</zorder>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="frame_error">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 61, 61);
border-radius: 8px;
position: absolute;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="label_error">
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Senha incorreta</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_closeError">
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	border-radius: 5px;
	background-image: url(:/Close/Images/X.png);
	background-position:center;
	background-color: rgb(255, 26, 26);

}

QPushButton:hover{
	background-color: rgb(50, 50, 50);
	color: rgb(200, 200, 200);
}
QPushButton:pressed{
	background-color: rgb(35, 35, 35);
	color: rgb(200, 200, 200);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="content">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(14, 102, 22);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="loginArea">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <widget class="QLineEdit" name="inputText_email">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>110</y>
           <width>250</width>
           <height>50</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	border: 2px solid rgb(14, 78, 20);
	border-radius: 5px ; 
	padding: 15px;
	background-color: rgb(254, 254, 254);
}

QLineEdit:hover{
	border:2px solid  rgb(254, 254, 254); 
}

QLineEdit:focus{
	color: rgb(000, 000, 000);
	border:2px solid  rgb(14, 78, 20); 
}</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>20</number>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Informe seu e-mail</string>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::LogicalMoveStyle</enum>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputText_dfName">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>170</y>
           <width>250</width>
           <height>50</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	border: 2px solid rgb(14, 78, 20);
	border-radius: 5px ; 
	padding: 15px;
	background-color: rgb(254, 254, 254);
}

QLineEdit:hover{
	border:2px solid  rgb(254, 254, 254); 
}

QLineEdit:focus{
	color: rgb(000, 000, 000);
	border:2px solid  rgb(14, 78, 20); 
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>100</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Informe o nome do banco</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_back">
         <property name="geometry">
          <rect>
           <x>45</x>
           <y>430</y>
           <width>160</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(254, 254, 254);
	font: 75 10pt &quot;Inter&quot;;
	color: rgb(11, 82, 18);
	border: 2px solid rgb(8, 50, 12);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgba(64, 155, 71, 250);
	font: 75 10pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border: 2px solid rgb(10, 65, 15);
	border-radius: 5px;
}

QPushButton:hover:pressed{
	background-color: rgb(24, 152, 35);
	font: 75 10pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border: 2px solid rgb(24, 152, 35);
	border-radius: 5px;
}

</string>
         </property>
         <property name="text">
          <string>VOLTAR AO MENU
ANTERIOR</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_addNewDf">
         <property name="geometry">
          <rect>
           <x>245</x>
           <y>430</y>
           <width>150</width>
           <height>51</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Inter</family>
           <pointsize>10</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(14, 78, 20);
	font: 75 10pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border: 2px solid rgb(8, 50, 12);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(17, 99, 24);
	font: 75 10pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border: 2px solid rgb(10, 65, 15);
	border-radius: 5px;
}

QPushButton:hover:pressed{
	background-color: rgb(24, 152, 35);
	font: 75 10pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border: 2px solid rgb(24, 152, 35);
	border-radius: 5px;
}

</string>
         </property>
         <property name="text">
          <string>ADICIONAR NOVO
BANCO</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Return</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="Title">
         <property name="geometry">
          <rect>
           <x>35</x>
           <y>10</y>
           <width>380</width>
           <height>66</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#ffffff;&quot;&gt;Adicione um novo &lt;br/&gt;banco de senhas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputText_dfPath">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>230</y>
           <width>174</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	border: 2px solid rgb(14, 78, 20);
	border-radius: 5px ; 
	padding: 15px;
	background-color: rgb(254, 254, 254);
}

QLineEdit:hover{
	border:2px solid  rgb(254, 254, 254); 
}

QLineEdit:focus{
	color: rgb(000, 000, 000);
	border:2px solid  rgb(14, 78, 20); 
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>100</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Local do banco</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_findPath">
         <property name="geometry">
          <rect>
           <x>275</x>
           <y>235</y>
           <width>75</width>
           <height>40</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(254, 254, 254);
	font: 75 11pt &quot;Inter&quot;;
	color: rgb(11, 82, 18);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgba(64, 155, 71, 250);
	font: 75 11pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border-radius: 5px;
}

QPushButton:hover:pressed{
	background-color: rgb(24, 152, 35);
	font: 75 11pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border-radius: 5px;
}

</string>
         </property>
         <property name="text">
          <string>Buscar...</string>
         </property>
        </widget>
        <widget class="QLabel" name="criptografia_title">
         <property name="geometry">
          <rect>
           <x>75</x>
           <y>290</y>
           <width>300</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;Chave de criptografia&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputText_encryptionKey">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>330</y>
           <width>175</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	border-radius: 5px ; 
	padding: 15px;
	background: #C4C4C4;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>100</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_copyKey">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>330</y>
           <width>75</width>
           <height>40</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(254, 254, 254);
	font: 75 11pt &quot;Inter&quot;;
	color: rgb(11, 82, 18);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgba(64, 155, 71, 250);
	font: 75 11pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border-radius: 5px;
}

QPushButton:hover:pressed{
	background-color: rgb(24, 152, 35);
	font: 75 11pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border-radius: 5px;
}

</string>
         </property>
         <property name="text">
          <string>Copiar</string>
         </property>
        </widget>
        <widget class="QLabel" name="criptografia_warnText">
         <property name="geometry">
          <rect>
           <x>92</x>
           <y>375</y>
           <width>261</width>
           <height>51</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Cuidado chave única!&lt;br/&gt;Não perca esta chave&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="QTImages.qrc"/>
 </resources>
 <connections/>
</ui>
